### ISSUE TITLE
Interval tool node flickers and crashes with multitouch

### ISSUE THREAD

**Issue by samreid** [phetsims/center-and-variability#225]
As described in https://github.com/phetsims/center-and-variability/issues/194, the Interval tool node flickers and crashes with multitouch.  In https://github.com/phetsims/center-and-variability/issues/194#issuecomment-1563651270 @marlitas said:

> Marking this for design, so that we can check in on multi-touch and see if we can move forward without it.

But maybe we can disable multitouch on it, and restore it if it turns out to be important.

**Comment by marlitas** [comment]
From a design check-in today:

@samreid wrote: "Today @catherinecarter agreed it is ok to exclude multitouch for translation. It's still nice to have multitouch on the handles though."

Transferring the TODO in that issue over here, and closing https://github.com/phetsims/center-and-variability/issues/194.

**Comment by samreid** [comment]
What about keyboard + mouse "multitouch"?

**Comment by samreid** [comment]
Describing the desired algorithm, as I understand it.

* When interaction starts (whether via keyboard/mouse/touch) on the IntervalToolNode itself (the body), make inputEnabled = false on both arrow handles. When interaction ends, restore inputEnable=true;
* When interaction starts on either arrow handle, make the intervalToolNode itself inputEnabled=false and vice versa.

**Comment by samreid** [comment]
In testing on the iPad, I observed that allowTouchSnag was missing from the prediction handles, and that it would be very helpful. So I added it.  This issue is at a good point for review/discussion.  The part that is getting complex/messy and could use refactoring or documentation is about all the isDragging/isKeyboardDragging/isMouseDragging properties.  I believe for sonification, the prediction arrows need to distinguish between mouse and keyboard.  If we implement #310 to be more discrete, we may need to implement it the same way.  In any case, this is ready for review/discussion.  I'll assign to @marlitas and @matthew-blackman to survey/review, but likely we should collaborate to get it working nicely.

**Comment by samreid** [comment]
Oops, it did have touch snag, but it felt like it didn't because the pointer are is tiny.  I'll open a separate issue for that.

**Comment by marlitas** [comment]
Sounds like this would be best to review synchronously.

**Comment by marlitas** [comment]
From review with @matthew-blackman today.

- [ ]  I observed that allowTouchSnag was missing from the prediction handles, and that it would be very helpful. So I added it. 
- [ ] The part that is getting complex/messy and could use refactoring or documentation is about all the isDragging/isKeyboardDragging/isMouseDragging properties. 
- [ ] I believe for sonification, the prediction arrows need to distinguish between mouse and keyboard.

**Comment by marlitas** [comment]
We looked at adding documentation around `isDragging,` `isKeyboardDragging`, and `isMouseDragging`. While doing so we also noticed that the predictionSlider was having to get a BooleanProperty passed in wether it was an interval tool handle or not. We decided to create `IntervalToolPredictionSlider` as a subclass as `PredictionSlider`. Both of us felt this was the right direction, but got stuck in an inheritance question. We left a TODO referencing this issue that we'd love some insight from @samreid on. 

Otherwise we have no further changes or recommendations for this issue.
